page-eight-two.php

```diff
@@ -0,0 +1,225 @@
+		</div>
+
+		<div class="row">
+			<!--Release Channels video-->
+                        <div class="col-md-6 text-center"><iframe width="100%" height="360" src="https://www.youtube.com/embed/RtNAx-PcZd8?rel=0" frameborder="0" allowfullscreen></iframe></div>
+                        <!--Introduction to Admin Panel video-->
+                        <div class="col-md-6 text-center"><iframe width="100%" height="360" src="https://www.youtube.com/embed/J2R-7lpBXjU?rel=0" frameborder="0" allowfullscreen></iframe></div>
+                        <!--Introducing the new experimental apps in the app store-->
+                        <div class="col-md-6 text-center"><iframe width="100%" height="360" src="https://www.youtube.com/embed/ujCFy41J2ag?rel=0" frameborder="0" allowfullscreen></iframe></div>
+                        <!--New external storage options explained-->
+                        <div class="col-md-6 text-center"><iframe width="100%" height="360" src="https://www.youtube.com/embed/nx5PGWc7X18?rel=0" frameborder="0" allowfullscreen></iframe></div>
+                </div>
+
+                <a name="developer"></a>
+		<div class="featurewrapper">
+			<h2>Improvements for Developers</h2>
+		</div>
+		ownCloud is <a href="http://karlitschek.de/2015/06/building-a-platform/">meant to be a platform</a>. This release introduces many improvements to the API, both to make more functionality available for app developers and to improve the existing interfaces.
+                        
+                <div class="row featurerow">
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="external_storage" src="<?php bloginfo('template_directory'); ?>/assets/img/features/external.png" />
+				<h3 class="featuretitle">Modular Authentication Mechanism Support</h3></br>
+				<p class="featuredesc">External storage backends are now built upon a 'BackendService' class, decoupling authentication from data handling. This reduces complexity in each external storage backend and makes writing new backends easier. It also makes the existing backends more flexible and capable of dealing with a wider range of authentication mechanisms than just 'username-password'.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="ldap" src="<?php bloginfo('template_directory'); ?>/assets/img/features/avatar.png" />
+				<h3 class="featuretitle">Capabilities and Query Building</h3></br>
+				<p class="featuredesc">A major change is the addition of a capabilities manager which allow apps to expose capabilities in a more generic way. App authors now also have access to a public API for the doctrine query builder.</p>
+			</div>
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="encryption" src="<?php bloginfo('template_directory'); ?>/assets/img/features/encryption.png" />
```

Comment:
Is this the intended thumbnail image here? "encryption" but the text below is more of a general topic.